legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -129,3 +130,30 @@ def post_businesses():
             HTTPStatus.SERVICE_UNAVAILABLE
 
     return ListFilingResource.put(bootstrap.identifier, None)
+
+
+@bp.route('/<string:identifier>', methods=['PATCH'])
+@cross_origin(origin='*')
+@jwt.requires_auth
+def update_businesses(identifier: str):
+    """Update a business."""
+    if not authorized(identifier, jwt, 'freeze_unfreeze'):
+        return jsonify({'message':
+                        f'You are not authorized to patch business {identifier}.'}), \
+                HTTPStatus.UNAUTHORIZED
+
+    business = Business.find_by_identifier(identifier)
+
+    if not business:
+        return jsonify({'message': f'{identifier} not found'}), HTTPStatus.NOT_FOUND
+
+    json_input = request.get_json()
+
+    admin_freeze = json_input['adminFreeze'] or False
+
+    business.admin_freeze = admin_freeze
+
+    business.save()
```

Comment:
Generally we don't change a businesses data through the API, it's done indirectly through a filing.
For instance when we apply a BN, a _Registrars Notation_ is added with a reason, so that all data changes are reflected in the ledger.
This breaks that whole pattern, and as such doesn't provide the customer with a reason it is frozen, or what they need to do next.

Reply:
Hi @thorwolpert we discussed about this in grooming couple of times. The freeze and unfreeze option from what I understand is an operation which should not appear in ledger, also it is communicated it is not a filing too. 

Documented in comments here: https://app.zenhub.com/workspaces/relationships-team-space-61435c47e483c4000f08e9f6/issues/bcgov/entity/13348

Happy to chat about this in standup.